Resources/Prototypes/_Moffstation/Catalog/uplink_catalog.yml

```diff
@@ -12,3 +11,19 @@
     Telecrystal: 8
   categories:
   - UplinkWeaponry
+
+- type: listing
+  id: UplinkHyperEutacticBlade
+  name: uplink-hypereutatic-blade-moff-name
+  description: uplink-hypereutatic-blade-moff-desc
+  icon:
+    sprite: /Textures/Objects/Weapons/Melee/hypereutactic_blade.rsi
+    state: icon 
+  discountCategory: veryRareDiscounts
+  discountDownTo:
+    Telecrystal: 8
+  productEntity: HyperEutacticBlade
+  cost:
+    Telecrystal: 14
+  categories:
+  - UplinkWeaponry
```

Comment:
```suggestion
  - UplinkWeaponry

```
The little "no entry" type thing indicates a missing end-of-file newline

<img width="248" height="125" alt="Image" src="https://github.com/user-attachments/assets/728eb170-1a9d-42cb-95ff-2aef3aaa3aad" />